platform/mellanox/issu-version/Makefile

```diff
@@ -19,12 +19,15 @@ SHELL = /bin/bash
 .SHELLFLAGS += -e
 
 MAIN_TARGET = issu-version
-BULLSEYE_FILES_PATH = /sonic/target/files/bullseye
+BUSTER_FILES_PATH = /sonic/target/files/buster
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+ifneq (,$(findstring buster,$(DEST)))
```

Comment:
Why this rule depends on Debian version?

Reply:
Not on the debian version but what is currently being built. 

If buster is being built it executes the program which exports the contents of the `issue_version` file in the buster slave docker. 

If bullseye is being built it doesn't have access to this utility so it copies from the buster targets (which has already been built) to itself. 

We do this so that we have the file available in both the buster and bullseye slave docker at build time and that both targets are correctly recognized by the build system and cached correctly.

Reply:
What is the reason of "If bullseye is being built it doesn't have access to this utility"? Any plan to have the utility (what is it?) in future?

I am wondering if you can make this Makefile not depends on any Debian versions, it should work for Buster, and any future Debian versions.

Reply:
You assume build order of Buster/Bullseye. Could you be explicit. I understand this PR is a temporary solution, and after SDK support Bullseye, all the Debian version dependency will be removed.